"""relationship error fix

Revision ID: 2785f4a2f98c
Revises: cba90f0849c1
Create Date: 2025-02-24 16:46:29.441618

"""
from typing import Sequence, Union

from alembic import op
import sqlalchemy as sa
import sqlmodel


# revision identifiers, used by Alembic.
revision: str = '2785f4a2f98c'
down_revision: Union[str, None] = 'cba90f0849c1'
branch_labels: Union[str, Sequence[str], None] = None
depends_on: Union[str, Sequence[str], None] = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_unique_constraint(None, 'notifications', ['uid'])
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_constraint(None, 'notifications', type_='unique')
    # ### end Alembic commands ###
